modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/utils/getIcon.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+const getIcon = ({symbol}: {symbol: string}): string => {
+	return `<svg xmlns="http://www.w3.org/2000/svg">
+		<use href="${Liferay.Icons.spritemap}#${symbol}" />
+	</svg>`;
+};
```

Comment:
Building icons from Clay spritemap. Something like this is needed for custom plugins, independent of considerations in https://github.com/liferay-frontend/liferay-portal/pull/5015#discussion_r2219420917